docs/cubeviz/plugins.rst

```diff
@@ -236,7 +202,7 @@ converted to velocity units before running the plugin if those units are
 desired for the output moment maps.
 
 Line or Continuum Maps
-======================
+----------------------
```

Comment:
Is this section level change intentional, @orifox ?

Reply:
Yes, I don't believe Line or Continuum Maps are at the Plugin level.